src/argowrapper/auth/utils.py

```diff
@@ -0,0 +1,29 @@
+import requests
+from argowrapper import logger
+from argowrapper.constants import (
+    COHORT_DEFINITION_BY_SOURCE_AND_TEAM_PROJECT_URL,
+)
+
+
+def get_cohort_ids_for_team_project(token, source_id, team_project):
+    header = {"Authorization": token, "cookie": "fence={}".format(token)}
+    url1 = "http://cohort-middleware-service/cohortdefinition-stats/by-source-id/{}/by-team-project?team-project={}".format(
```

Comment:
@tianj7 is this needed or is this just some leftover cruft that needs to be cleaned up?

Reply:
thanks for catching this, removed.